app/scripts/HeatmapTiledPixiTrack.js

```diff
@@ -269,19 +289,35 @@ class HeatmapTiledPixiTrack extends TiledPixiTrack {
   }
 
   updateValueScale() {
+    let minValue = this.minValue();
+    let maxValue = this.maxValue();
+
+    // There might be only one value in the visible area. We extend the
+    // valuescale artificially, so that point is still displayed
+    const epsilon = 1e-6;
+    if (
+      minValue !== undefined &&
+      minValue !== null &&
+      maxValue !== undefined &&
+      maxValue !== null &&
+      Math.abs(minValue - maxValue) < epsilon
+    ) {
+      // don't go to or below 0 in case there is a log scale
+      const offset = 1e-3;
```

Comment:
This is just relevant, when `minValue === maxValue`, e.g. when there is only one data point in view. We then artificially make the valueScale a little larger, so that pixel still gets displayed.